src/ape/api/transactions.py

```diff
@@ -27,6 +27,7 @@ class TransactionAPI(BaseInterfaceModel):
     """
 
     chain_id: int = Field(0, alias="chainId")
```

Comment:
I would really like to pull the changes to `TransactionAPI` out of this PR and move them over to #970

Reply:
agreed, I think since we consider this feature in beta and there are solid opt-in behavior with purging to reset, that de-risks the changes you need to make to these models